conformance/packages/conformance-tests/src/resolver/dnssec/scenarios/secure.rs

```diff
@@ -315,3 +316,67 @@ fn nxdomain_nsec3() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn no_root_ds_query() -> Result<()> {
+    let network = Network::new()?;
+
+    let signed_root_ns = NameServer::new(&PEER, FQDN::ROOT, &network)?;
+    let signed_root_ns = signed_root_ns.sign(SignSettings::default())?;
+    let trust_anchor = signed_root_ns.trust_anchor();
+    drop(signed_root_ns);
+
+    let mut root_ns = NameServer::new(&PEER, FQDN::ROOT, &network)?;
+    let tld_ns = NameServer::new(&PEER, FQDN::TEST_TLD, &network)?;
+
+    root_ns.referral_nameserver(&tld_ns);
+
+    let root_ns = root_ns.start()?;
+    let _tld_ns = tld_ns.start()?;
+
+    let root = root_ns.root_hint();
+    let resolver = Resolver::new(&network, root)
+        .trust_anchor(&trust_anchor)
+        .start()?;
+
+    let mut tshark = resolver.eavesdrop()?;
+
+    let client = Client::new(&network)?;
+    client.dig(
+        *DigSettings::default().recurse().authentic_data(),
+        resolver.ipv4_addr(),
+        RecordType::TXT,
+        &FQDN::TEST_TLD,
+    )?;
+
+    let client_ip = client.ipv4_addr();
+    tshark.wait_until(
+        |captures| {
+            captures.iter().any(|capture| {
+                matches!(
+                    capture,
+                    Capture {
+                        direction: Direction::Outgoing { destination },
+                        ..
+                    } if *destination==client_ip
```

Comment:
Nit: I guess rustfmt missed it because this is inside a macro call, but please add spaces around `==`?